questions/qa-bidi-unicode-controls.en.html

```diff
@@ -92,19 +99,19 @@ <h2>Changing the base direction</h2>
           <th>Equivalent markup</th>
           <th>Notes</th>
         </tr>
-        <tr>
+      <tr>
           <td>LRI</td>
           <td>LEFT-TO-RIGHT ISOLATE</td>
           <td>U+2066</td>
           <td>dir = &quot;ltr&quot;</td>
-          <td>sets base direction to LTR and isolates the embedded content from the surrounding text</td>
-        </tr>
+          <td>ditto, but for LTR</td>
```

Comment:
It says "ditto" but it's the first row in the table. Maybe we should swap it with the row below?

Reply:
Thanks for spotting that. Fixed.